### DISCUSSION TITLE
Photo synch options (to prevent large amounts of other users photos being synched to all other devices)

### DISCUSSION THREAD

**Discussion by timbo-barnes** [opengisch/QField#5393] [Ideas]
We're currently using QField with Qfield cloud to carry out field surveys with a large number of users surveying a large number of points. Each point surveyed requires collection of up to 4 photos. As time goes on, the synchronization burden becomes too heavy as each user needs to sync both new point data AND all other users' photos. Even when setting a max resolution, this is growing to a very large amount of data to download each time.
To get around this, and allow just pushing of a users photos to Qfield Cloud, it'd be great to set an option to set the attachment directory to be a 'one way' synch. This would mean that each user can take their own photos but not be forced to download other users photos, greatly reducing synchronization file sizes.

**Comment by meyerlor** [comment]
I just found this platform, where apparently enhancement ideas are collected and voted for: https://ideas.qfield.org/

Dunno why there is no advertisment or any official link from github/the website pointing to this page, but maybe post your idea there? :)
